src/flow-spec/IBlock.ts

```diff
@@ -49,6 +49,7 @@ export interface IBlockUIMetadataCanvasCoordinates {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export interface IBlockUIMetadata extends Record<string, any> {
   canvas_coordinates: IBlockUIMetadataCanvasCoordinates
+  should_auto_update_name: boolean
```

Comment:
- [ ] @object-required , don't forget to bump the sub-version, and publish to npm ;)

Reply:
@object-required , Shouldn't we put this as optional ? I think we should, that will avoid us having updating old saved flows which don't have it.
- [ ] make optional